README.md

```diff
@@ -47,6 +47,12 @@ the context after the span finished.
 
 LogRecord is the simple builder for the LogData, which is the Log format of Skywalking.
 
+## Metrics
+
+### MeterRecord
+
+MeterRecord is the simple builder for the MeterData, which is the metrics format of Skywalking.
```

Comment:
You should provide a full list of the metric data types with explanations.

We have a short version on main repo docs, https://skywalking.apache.org/docs/main/latest/en/protocols/readme/#metrics which targets to proto file.

Reply:
Updated.